refinery/tool_manager/models.py

```diff
@@ -87,7 +87,7 @@ def cast_param_value_to_proper_type(self, parameter_value):
         if self.value_type in self.STRING_TYPES:
             return str(parameter_value)
         elif self.value_type == self.BOOLEAN:
-            return bool(parameter_value)
+            return True if parameter_value.lower() == "true" else False
```

Comment:
You don't need the bracketing if-then... but is this the right direction? I had thought we were moving towards allowing native type, rather than away?